testing/src/cypress/e2e/04-api-institution/01-instition-dashboard.cy.ts

```diff
@@ -0,0 +1,150 @@
+      method: "GET",
+      url: `${BASE_URL}/institution-user/my-details`,
+      followRedirect: false,
+      headers: {
+        "Content-Type": "text/html",
+        Authorization: TOKEN,
+      },
+    }).then((resp) => {
+      expect(resp.status).to.be.equal(200);
+      expect(resp.body.user).to.have.property("userName", USER_F.userName);
+      expect(resp.body.user).to.have.property("firstName", USER_F.firstName);
+      expect(resp.body.user).to.have.property("lastName", USER_F.lastName);
+      expect(resp.body.user).to.have.property("isActive", true);
+      expect(resp.body.user).to.have.property(
+        "userFullName",
+        USER_F.userFullName
+      );
+      expect(resp.body.user).to.have.property("isAdmin", true);
+      expect(resp.body.user).to.have.property("email", USER_F.email);
+      expect(resp.body.authorizations).to.have.property(
+        "institutionId",
+        USER_F.authorizations.institutionId
+      );
+      expect(resp.body.authorizations.authorizations[0]).to.have.property(
+        "locationId",
+        USER_F.authorizations.authorizations[0].locationId
+      );
+      expect(resp.body.authorizations.authorizations[0]).to.have.property(
+        "userType",
+        USER_F.authorizations.authorizations[0].userType
+      );
+      expect(resp.body.authorizations.authorizations[0]).to.have.property(
+        "userRole",
+        USER_F.authorizations.authorizations[0].userRole
+      );
+    });
+  });
+
+  it("Get institution details", () => {
+    cy.request({
+      method: "GET",
+      url: `${BASE_URL}/institution`,
+      followRedirect: true,
+      headers: {
+        "Content-Type": "text/html",
+        Authorization: TOKEN,
+      },
+    }).then((resp) => {
+      const body = resp.body;
+      const addr = body.mailingAddress;
+      expect(resp.status).to.be.equal(200);
+      expect(body).to.have.property(
+        "establishedDate",
+        COLLEGE_F.establishedDate
+      );
+
+      expect(body).to.have.property(
+        "formattedEstablishedDate",
+        COLLEGE_F.formattedEstablishedDate
+      );
+      expect(body).to.have.property(
+        "hasBusinessGuid",
+        COLLEGE_F.hasBusinessGuid
+      );
+      expect(body).to.have.property(
+        "institutionType",
+        COLLEGE_F.institutionType
+      );
+      expect(body).to.have.property(
+        "institutionTypeName",
+        COLLEGE_F.institutionTypeName
+      );
+      expect(body).to.have.property("isBCPrivate", COLLEGE_F.isBCPrivate);
+      expect(body).to.have.property(
+        "legalOperatingName",
+        COLLEGE_F.legalOperatingName
+      );
+      expect(addr).to.have.property(
+        "addressLine1",
+        COLLEGE_F.mailingAddress.addressLine1
+      );
+      expect(addr).to.have.property(
+        "addressLine2",
+        COLLEGE_F.mailingAddress.addressLine2
+      );
+      expect(addr).to.have.property(
+        "canadaPostalCode",
+        COLLEGE_F.mailingAddress.canadaPostalCode
```

Comment:
fyi, if the institution country is not `Canada`. then the postal code  is also different 
![image](https://user-images.githubusercontent.com/77353155/187500921-b2829315-706d-4057-b69c-4e3de1d969bb.png)

Reply:
![image](https://user-images.githubusercontent.com/77353155/187501589-2c5aadc6-cc1b-4444-bdac-dde967190cac.png)
 is this other postal code ?

Reply:
Sure, this is handled in the expected results. So if the specific institution/location is outside CA, then the respective expected results will be reflecting the same. If the location does not have postal code, a null be used

Reply:
As discussed, will add a separate test case once we have login details